### ISSUE TITLE
Add config option for assets that are hosted at different domain 

### ISSUE THREAD

**Issue by stuntbox** [robwierzbowski/jekyll-picture-tag#129]
Putting in a feature request for something like a `cdn_url` option for sites that host image assets on a different subdomain, such as an S3/CloudFront setup. In the past I’ve hacked this into my own install of this plugin, and can take a crack at putting together a PR.

**Comment by rbuchberger** [comment]
I'll see what I can do, it shouldn't be too hard.

**Comment by stuntbox** [comment]
Needed this for a project, so I threw together a quick [PR](https://github.com/robwierzbowski/jekyll-picture-tag/pull/130). I don’t know my way around Ruby or gems that well, so definitely happy for any suggestions on smarter ways to do this! 

I added a default `cdn_url` value (empty string) to `global.yml`, and a simple if statement inside where `url_prefix` is defined in `configuration.rb`. I’m making an assumption that a CDN value will always be an absolute URL, so it checks for the CDN value first and skips the `relative_url` option if it’s present. 

Updated the README to reflect the new option. Let me know what you think!
